src/test/config.sol

```diff
@@ -1370,6 +1370,96 @@ contract Config {
             calc_cut:     0,
             lerp:         false
         });
+        afterSpell.collaterals["RWA007-A"] = CollateralValues({
+            aL_enabled:   false,
+            aL_line:      0,
+            aL_gap:       0,
+            aL_ttl:       0,
+            line:         1_000_000,
+            dust:         0,
+            pct:          0,
+            mat:          10000,
```

Comment:
```suggestion
            mat:          100_00,
```

Reply:
This should be changed everywhere or just left alone for consistency imo.